web/themes/custom/usagov/templates/html.html.twig

```diff
@@ -276,7 +281,7 @@ const scrollFunc = () => {
   } else {
     scrollToTopButton.className = "";
   }
-};
+%};
```

Comment:
Typo here -- % doesn't belong. 
This actually is causing a syntax error and causing the "Top" button not to appear when I scroll down the home page